src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -146,15 +150,32 @@ class Monitoring final
                                         }
                                         else
                                         {
+                                            logWarn(IC_MONITORING,
+                                                    "Cluster health status: '%s'",
+                                                    Utils::toUpperCase(fields.at(HealthCheckColumns::STATUS)).c_str());
                                             value.second = false;
                                         }
                                     }
                                     else
                                     {
+                                        logDebug2(IC_MONITORING,
+                                                  "Bad response from _cluster/health: %s",
+                                                  originalResponse.c_str());
                                         value.second = false; // LCOV_EXCL_LINE
                                     }
                                 },
-                                [&](const std::string& error, const long /*statusCode*/) { value.second = false; },
+                                [&](const std::string& error, const long statusCode)
+                                {
+                                    std::string errorMessage = error;
+                                    constexpr auto NOT_USED {-1};
+
+                                    if (statusCode != NOT_USED)
```

Comment:
Question: why we avoid logging the error message in case `statusCode` is `NOT_USED` ?

Reply:
I left a comment on the issue about that. For it was enough with the log in ServerSelector.

Reply:
I understand.
Thank you.